### ISSUE TITLE
Update "Developing on a CISA laptop" document

### ISSUE THREAD

**Issue by vickyszuchin** [cisagov/manage.get.gov#2325]
### Issue description

A new federal .gov team member should be able to use their government-furnished equipment (GFE) for development (and similar) work. We have outlined the broad steps for setup in "[Developing on a CISA laptop](https://docs.google.com/document/d/1ZyaaTWOEgVEHiHarNuv97CnYfUFe-4pfbuQm4ocLkn4/edit#heading=h.nffo9qwynuw6)".

Members of the .gov team have reviewed and stepped through the instructions (#2321) to make sure that the document is accurate. They have identified some improvements to the document.

### Acceptance criteria

- [ ] Review suggested improvements to document and determine if they are valid
- [ ] Update the document as needed
- [ ] Create an issue to have an engineer with a PIV and a CISA laptop to attempt these steps, prioritized for next sprint.

### Links to other issues

Blocked by:

- #2321

**Comment by vickyszuchin** [comment]
@abroddrick update ticket as you see fit. I have slotted it tentatively for sprint 49 as requested.

**Comment by h-m-f-t** [comment]
Closing in favor of #2492.
